src/User/Module.php

```diff
@@ -81,6 +81,10 @@ class Module extends BaseModule
      * @var bool whether to enable two factor authentication or not
      */
     public $enableTwoFactorAuthentication = false;
+     /**
+     * @var array list of permissions for which two factor authentication is mandatory
+     */
+    public $twoFactorAuthenticationForcedPermissions = ['admin'];
```

Comment:
This change is disruptive for upgrades, we shouldn't enforce anyone, the default must be empty